bakery/service.go

```diff
@@ -79,8 +93,27 @@ func NewService(p NewServiceParams) (*Service, error) {
 	return svc, nil
 }
 
+// WithRootKeyStore returns a copy of service where macaroon creation and
+// lookup uses the given store to look up macaroon root keys
```

Comment:
given root key store. Service has a Store attribute so I'd like to be explicit here